Jenkinsfile

```
@@ -1,5 +1,9 @@
 properties([disableConcurrentBuilds(abortPrevious: true), buildDiscarder(logRotator(numToKeepStr: '7'))])
 
+if (BRANCH_NAME == 'master' && currentBuild.buildCauses*._class == ['jenkins.branch.BranchEventCause']) {
```

```suggestion
if (BRANCH_IS_PRIMARY && currentBuild.buildCauses*._class == ['jenkins.branch.BranchEventCause']) {
```
(optional nit)

---

```suggestion
if (BRANCH_IS_PRIMARY as boolean && currentBuild.buildCauses*._class == ['jenkins.branch.BranchEventCause']) {
```
perhaps (it will be a `String`). Might work but I would actually prefer to specifically suppress builds on `master`; in the unlikely event someone creates some sort of backport branch, we can deal with the automatic builds there.